10-inference.Rmd

```diff
@@ -0,0 +1,806 @@
+# Introduction to Statistical Inference {#inference}
+
+```{r 11-setup, include= F}
+library(tidyverse) 
+require(grid) #  grid.bezier()
+
+max_count <- function(dist) {
+  max(ggplot_build(dist)$data[[1]]$count)
+}
+
+max_x <- function(dist){ 
+  max(ggplot_build(dist)$data[[1]]$xmax)
+}
+
+min_x <- function(dist) {
+  ggp_data <- ggplot_build(dist)
+  min(ggp_data$data[[1]]$xmin)
+}
+```
+
+
+## Overview
+In almost all data analysis tasks in practice, we want to draw conclusions about some unknown
+aspect of a population of interest based on observed data sampled from that
+population; we typically do not get data on the *full* population.
```

Comment:
I would say this is true mainly for inferential questions. In predictive questions, we are interested mainly in predicting future values for individuals, or what features are important for predicting future values for individuals. Maybe soften this statement to "Another common data analysis task in practice is to draw conclusions...

```suggestion
Another common data analysis task in practice is to draw conclusions about some unknown
aspect of a population of interest based on observed data sampled from that
population; we typically do not get data on the *entire* population.
```